source/guides/getting-started.md

```diff
@@ -159,6 +159,16 @@ Note that, although Hanami is ready for a Behavior Driven Development workflow o
 We'll go with [Minitest](https://github.com/seattlerb/minitest) here (which is the default), but we can use [RSpec](http://rspec.info) by creating the project with `--test=rspec` option.
 Hanami will then generate helpers and stub files for it.
 
+Nonetheless we'll have to setup the test database to be able to run the tests.
+First we have to define our database url in `.env.test`.
```

Comment:
Yup you're right, it already exists. What I wanted to say is that you'll maybe want to tweak it. The default is: `"postgresql://localhost/shopit_api_test"` but some people do set a password on it, even if it's a local development DB.